### DISCUSSION TITLE
Ant Media Server 2.12.0 release is available with new features and improvements :tada:

### DISCUSSION THREAD

**Discussion by burak-58** [ant-media/Ant-Media-Server#6902] [Announcements]
Here are the key features for 2.12.0 release:

- Update FFmpeg to 7.1
- Support WebP, JPEG previews
- Support ID3 in HLS HEVC streams
- Support USB stick licensing
- Optimize MongoDB utilization

You can check whole change logs [here](https://github.com/ant-media/Ant-Media-Server/releases/tag/ams-v2.12.0). 

Enjoy with it :heart:

**Comment by IVEN2024** [comment]
is it safe to update to this version ?? Enterprise edition ??

**Reply by Mohit-3196** [reply]
@IVEN2024,
Yes, it is.
